### PR TITLE
ultra senior titles

### PR DESCRIPTION
**Pull request by BootlegBow**
## About The Pull Request

adds ultra senior job titles

## Why It's Good For The Game

shitposting

## Changelog
:cl: Jones
add: Senior titles, unlockable after 100 hours of any job. The job titles are mainly memey. Engineers can be called bob the builder now, world peace achieved.
/:cl:

### CODE REVIEW

**Comment by retlaw34** [comment]
Im sorry these are great, but they are too memey

**Comment by JRDead** [comment]
atmos should be atmosian

**Comment by BootlegBow** [comment]
> Im sorry these are great, but they are too memey

image

**Review by JRDead — APPROVED** [review]
lovely

**Comment by Kitsunemitsu** [comment]
I'll give this a tentative yes.
I will need to look at the code, but I am a-okay with shitposty names for people with 100+ hours in a job

**Comment by DaddyIssues98** [comment]
i dont know what im doing with the reviews, sorry

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -1202,6 +1202,9 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 				if(user.client.prefs.exp[job_title] >= CONFIG_GET(number/senior_timelock)) //If they have more than 50 hours (300 Minutes) past the required time needed for the job, give them access to the senior title
 					for(var/g in J.senior_title)
 						titles_list += g
+				if(user.client.prefs.exp[job_title] >= 600)//100 hours, i can't be fucked making this a config
```
Comment (EgorDinamit) [inline code/modules/client/preferences.dm:1205]: Make it a config.

File: code/modules/client/preferences.dm [hunk 2]
```diff
@@ -1202,6 +1202,9 @@ GLOBAL_LIST_EMPTY(preferences_datums)
...
+					for(var/g in J.ultra_senior_title)
+						titles_list += g
```
Comment (EgorDinamit) [inline code/modules/client/preferences.dm:1207]: ultra_senior title is not a list, instead do this:
```suggestion
					if(J.ultra_senior_title)
						titles_list += J.ultra_senior_title
```
Same for senior_title.

File: code/modules/client/preferences.dm (hunk 1 above)
Comment (BootlegBow) [reply code/modules/client/preferences.dm:1205]: fiiiiiiiiiine

File: code/modules/client/preferences.dm (hunk 2 above)
Comment (BootlegBow) [reply code/modules/client/preferences.dm:1207]: senior_title is a list here, some roles have multiple

File: ModularBungalow/bungalowconfig.dm [hunk 3]
```diff
@@ -1,3 +1,7 @@
 /datum/config_entry/number/senior_timelock
 	config_entry_value = 3000
 	min_val = 0
+
+/datum/config_entry/number/ultra_senior_timelock
+	config_entry_value = 6000
+	min_val - 3000
```
Comment (EgorDinamit) [inline ModularBungalow/bungalowconfig.dm:7]: Seriously..?

**Comment by bruhlookatthisdood** [comment]
no ai super senior title of "Omnissiah" I am very angry now ):<

**Comment by bruhlookatthisdood** [comment]
is this still alive

**Comment by bruhlookatthisdood** [comment]
this is my pr now

**Comment by SqueezyS** [comment]
Based.

**Review by Kitsunemitsu — APPROVED** [review]
Everything looks good.

I personally don't agree with them all being shitposts, but that's a road we can get across later.

**Comment by bruhlookatthisdood** [comment]
this PR is on life support
if we go full mrp the wire is gonna get cut

**Comment by Kitsunemitsu** [comment]
Has a fix for Alt-Titles I didn't want to make modular.
These are fine, for now. 
If someone complains we will change it.

**Comment by bruhlookatthisdood** [comment]
it happened guys
this pr is like 18 years old

**Pull request merged**